<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="0" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-05-15T01:26:26 MDT" name="Default suite" finished-at="2020-05-15T01:26:26 MDT" duration-ms="75">
    <groups>
    </groups>
    <test started-at="2020-05-15T01:26:26 MDT" name="Default test" finished-at="2020-05-15T01:26:26 MDT" duration-ms="75">
      <class name="com.yorbit.tests.NewUserRegistration">
        <test-method is-config="true" signature="startServer()[pri:0, instance:com.yorbit.tests.NewUserRegistration@5ccddd20]" started-at="2020-05-15T01:26:26 MDT" name="startServer" finished-at="2020-05-15T01:26:26 MDT" duration-ms="122" status="FAIL">
          <exception class="io.appium.java_client.service.local.InvalidServerInstanceException">
            <message>
              <![CDATA[NodeJS is either not installed or its executable not present in PATH]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.appium.java_client.service.local.InvalidServerInstanceException: NodeJS is either not installed or its executable not present in PATH
at io.appium.java_client.service.local.AppiumServiceBuilder.validatePath(AppiumServiceBuilder.java:127)
at io.appium.java_client.service.local.AppiumServiceBuilder.findBinary(AppiumServiceBuilder.java:137)
at io.appium.java_client.service.local.AppiumServiceBuilder.findDefaultExecutable(AppiumServiceBuilder.java:180)
at org.openqa.selenium.remote.service.DriverService$Builder.build(DriverService.java:355)
at io.appium.java_client.service.local.AppiumDriverLocalService.buildService(AppiumDriverLocalService.java:87)
at com.yorbit.tests.BaseTest.startServer(BaseTest.java:192)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:64)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:364)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:318)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:327)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- io.appium.java_client.service.local.InvalidServerInstanceException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startServer -->
        <test-method is-config="true" signature="startReport()[pri:0, instance:com.yorbit.tests.NewUserRegistration@5ccddd20]" started-at="2020-05-15T01:26:26 MDT" name="startReport" finished-at="1969-12-31T17:00:00 MST" duration-ms="-1589527586453" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startReport -->
        <test-method signature="registerNewUserAndVerify()[pri:0, instance:com.yorbit.tests.NewUserRegistration@5ccddd20]" started-at="2020-05-15T01:26:26 MDT" name="registerNewUserAndVerify" finished-at="2020-05-15T01:26:26 MDT" duration-ms="0" status="SKIP">
          <exception class="io.appium.java_client.service.local.InvalidServerInstanceException">
            <message>
              <![CDATA[NodeJS is either not installed or its executable not present in PATH]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.appium.java_client.service.local.InvalidServerInstanceException: NodeJS is either not installed or its executable not present in PATH
at io.appium.java_client.service.local.AppiumServiceBuilder.validatePath(AppiumServiceBuilder.java:127)
at io.appium.java_client.service.local.AppiumServiceBuilder.findBinary(AppiumServiceBuilder.java:137)
at io.appium.java_client.service.local.AppiumServiceBuilder.findDefaultExecutable(AppiumServiceBuilder.java:180)
at org.openqa.selenium.remote.service.DriverService$Builder.build(DriverService.java:355)
at io.appium.java_client.service.local.AppiumDriverLocalService.buildService(AppiumDriverLocalService.java:87)
at com.yorbit.tests.BaseTest.startServer(BaseTest.java:192)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:64)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:364)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:318)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:327)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- io.appium.java_client.service.local.InvalidServerInstanceException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registerNewUserAndVerify -->
        <test-method is-config="true" signature="getResult(org.testng.ITestResult)[pri:0, instance:com.yorbit.tests.NewUserRegistration@5ccddd20]" started-at="2020-05-15T01:26:26 MDT" name="getResult" finished-at="1969-12-31T17:00:00 MST" duration-ms="-1589527586516" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method is-config="true" signature="endReport()[pri:0, instance:com.yorbit.tests.NewUserRegistration@5ccddd20]" started-at="2020-05-15T01:26:26 MDT" name="endReport" finished-at="1969-12-31T17:00:00 MST" duration-ms="-1589527586522" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
        <test-method is-config="true" signature="stopServer()[pri:0, instance:com.yorbit.tests.NewUserRegistration@5ccddd20]" started-at="2020-05-15T01:26:26 MDT" name="stopServer" finished-at="1969-12-31T17:00:00 MST" duration-ms="-1589527586692" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- stopServer -->
      </class> <!-- com.yorbit.tests.NewUserRegistration -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
